### PR TITLE
docs(UI): Added info to landing page

### LINKED ISSUE #39: Clean README in repo

### PR DESCRIPTION
**Pull request by fcossio**
Closes #39

### CODE REVIEW

**Comment by fcossio** [comment]
Hey @yaelsuarez could you give me feedback of the info in the landing page please?

File: README.md [hunk 1]
```diff
@@ -1,32 +1,51 @@
...
 ### Online
 
-[fiberdiameter.com](https://www.fiberdiameter.com)
+The app is openly available at [FiberDiameter.com](https://www.fiberdiameter.com)
+
+You can start by playing with the test image. First select the wand tool and then select the fiber that you want to measure.
+
+The deep learning models are run directly in the browser. This means that your images never leave your computer and your intellectual property is preserved.
+
+In the world of machine learning more data translates to better performance of the models. If you would like to donate images for training models, please get in touch with YaelSuarez. This will make the models better for you and for everyone.
```
Comment (yaelsuarez) [inline README.md]: Add link to YaelSuarez (yael.suarez@farmaci.uu.se)

File: frontend/pages/index.tsx [hunk 2]
```diff
@@ -30,21 +31,28 @@ const Home: NextPage = () => {
...
-          A deep learning-assisted app to{" "}
-          <span className='underline decoration-2 decoration-white-500'>
-            measure
-          </span>{" "}
-          your{" "}
-          <span className='underline decoration-2 decoration-white-500'>
-            fibers
-          </span>
+        <h2 className="h2-center">
+          Measure the diameter of fibers in microscopy images<br/>
```
Comment (yaelsuarez) [inline frontend/pages/index.tsx]: electron microscopy != microscopy 
Maybe better to put "Measure the diameter of fibers in electron microscopy images"

File: frontend/pages/index.tsx [hunk 3]
```diff
@@ -30,21 +31,28 @@ const Home: NextPage = () => {
...
+          About
+        </h2>
+        <p>
+          In the app, you can start by playing with the test image. First select the wand tool and
+          then select the fiber that you want to measure.
+          The deep learning models are run directly in the browser. This means that your
+          images never leave your computer and your intellectual property is preserved.
+          In the world of machine learning more data translates to better performance of
+          the models. If you would like to donate images for training models, please get
+          in touch with YaelSuarez. This will help us ship better models for everyone.
```
Comment (yaelsuarez) [inline frontend/pages/index.tsx]: Add link to mail of YaelSuarez (yael.suarez@farmaci.uu.se)

File: frontend/pages/index.tsx [hunk 4]
```diff
@@ -30,21 +31,28 @@ const Home: NextPage = () => {
...
         </Link>
+        <h2 style={{marginTop:"50px"}}>
+          About
+        </h2>
+        <p>
+          In the app, you can start by playing with the test image. First select the wand tool and
+          then select the fiber that you want to measure.
+          The deep learning models are run directly in the browser. This means that your
+          images never leave your computer and your intellectual property is preserved.
+          In the world of machine learning more data translates to better performance of
```
Comment (yaelsuarez) [inline frontend/pages/index.tsx]: In the world of machine learning, more data....

**Review by yaelsuarez — COMMENTED** [review]
Great modifications!

**Pull request merged**